byterun/unix.c

```diff
@@ -80,9 +81,18 @@ int caml_write_fd(int fd, int flags, void * buf, int n)
 {
   int retcode;
  again:
+#if defined(NATIVE_CODE) && defined(WITH_SPACETIME)
+  if ((flags & CHANNEL_FLAG_BLOCKING_WRITE)
+      == CHANNEL_FLAG_BLOCKING_WRITE) {
```

Comment:
I wonder if it's better style in general to write the equality test, then you don't need to worry...